distribution/docs/src/main/resources/_contents/_configuring/configuring-from-configuration-files-contents.adoc

```diff
@@ -0,0 +1,746 @@
+
+
+[IMPORTANT]
+====
+Since certain bundles can only be configured using the `.config` file format, this file format should be used.
+====
+
+[WARNING]
+====
+Only root can access ports < 1024 on Unix systems.
+For suggested ways to run ${branding} with ports < 1024 see <<How do I use port 80 as a non-root user?>>.
+====
+
+==== Configuring ${branding} config Files
+
+The ${branding} is configured using `.config` files.
+Like the Karaf `.cfg` files, these configuration files must be located in the `<${ddf-branding}_HOME>/etc/` directory, have a name that matches the _configuration persistence ID_ (PID) they represent, and have a `service.pid` property set to the configuration PID.
+
+As opposed to `.cfg` however, this type of configuration file supports lists within configuration values (metatype `cardinality` attribute greater than 1).
+
+[IMPORTANT]
+====
+This new configuration file format *must* be used for any configuration that makes use of lists.
+Examples include Web Context Policy Manager (PID: `org.codice.ddf.security.policy.context.impl.PolicyManager`)
+and Security STS Guest Claims Handler (PID: `ddf.security.sts.guestclaims`).
+====
+
+[WARNING]
+====
+Only one configuration file should exist for any given PID.
+The result of having both a `.cfg` and a `.config` file for the same PID is undefined and could cause the application to fail.
+====
+
+The main purpose of the configuration files is to allow administrators to pre-configure ${branding} without having to use the ${admin-console}.
+In order to do so, the configuration files need to be copied to the `<${branding}_HOME>/etc` directory after ${branding} zip has been extracted.
+
+Upon start up, all the `.config` files located in `<${branding}_HOME>/etc` are automatically read and processed.
+Files that have been processed successfully are moved to `<${branding}_HOME>/etc/processed` so they will not be processed again when the system is restarted.
+Files that could not be processed are moved to the `<${branding}_HOME>/etc/failed` directory.
+
+${branding} also monitors the `<${branding}_HOME>/etc` directory for any new `.config` file that gets added.
+As soon as a new file is detected, it is read, processed and moved to the appropriate directory based on whether it was successfully processed or not.
+
+==== Configuring Managed Service Factory Bundles
+
+Services that are created using a Managed Service Factory can be configured using `.config` files as well.
+The configuration files follow a different naming convention however.
+The files must start with the Managed Service Factory PID, be followed by a unique identifier and have a `.config` extension.
+For instance, assuming that the Managed Service Factory PID is `org.codice.ddf.factory.pid` and two instances of the service need to be configured, files `org.codice.ddf.factory.pid.uniqueID1.config` and `org.codice.ddf.factory.pid.uniqueID2.config` should be created and added to `<${branding}_HOME>/etc`.
```

Comment:
Wonder if we should replace `uniqueID1` and `uniqueID2` with `<unique ID 1>` and `<unique ID 2>` to make it clear that those can be anything.